makefile

```diff
@@ -1,7 +1,7 @@
 # start project configuration
 name := gimlet
 buildDir := build
-packages := $(name) ldap
+packages := $(name) ldap prebuilt rolemanager
```

Comment:
prebuilt? can we have an ACL package for this kind of thing?

Reply:
changed to acl